apps/dapp/src/app/layout.tsx

```diff
@@ -1,37 +1,48 @@
-import { ProfileContextProvider } from '@/context/ProfileContext';
-import type { Metadata } from 'next';
-import { Barlow } from 'next/font/google';
-import WagmiWrapper from '../components/WagmiWrapper';
-
-import { Toaster } from '@/components/Ui/Toaster';
-import './globals.css';
-
-const barlow = Barlow({
-  subsets: ['latin'],
-  weight: ['400', '600', '700', '800'],
-  style: ['normal', 'italic'],
-});
-
-export const metadata: Metadata = {
-  title: 'EVM Profile',
-  description: 'Create EVM compatible Profile tokens',
-};
-
-export default function RootLayout({
-  children,
-}: {
-  children: React.ReactNode;
-}) {
-  return (
-    <html lang="en">
-      <body className={`${barlow.className}`}>
-        <WagmiWrapper>
-          <ProfileContextProvider>
-            {children}
-            <Toaster />
-          </ProfileContextProvider>
-        </WagmiWrapper>
-      </body>
-    </html>
-  );
-}
+import './globals.css';
+
+import { Toaster } from '@/ui/UiToaster';
+// import { AudioPlayerGlobal } from '@/components/AudioPlayerGlobal/AudioPlayerGlobal';
+import { Barlow } from 'next/font/google';
+import WagmiWrapper from '../components/WagmiWrapper';
+// import RQProvider from './components/RQProvider';
+// import { GoogleAnalytics } from '@/components/GoogleAnalytics';
+
+const barlow = Barlow({
+  subsets: ['latin'],
+  weight: ['400', '600', '700', '800'],
+  style: ['normal', 'italic'],
+});
+
+export const metadata = {
+  title: 'DROP.',
+  description:
+    "Drop tracks, Build fans. Deluxe Releases from Electronic Music's Finest",
+};
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en" className="antialiased">
+      {/* <head>

```

Comment:
Let's clean up the comments, or implement the commented out code (I think the latter is prob better). We can then instruct users to add the env var in the readme / env example file.